jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -471,6 +476,45 @@ private static Path getCheckedCacheDirectory(String property, Path defaultPath)
     return defaultPath;
   }
 
+  /**
+   * Checks whether or not the known default Java 8 distroless base image is being used. Checking
+   * against only images known to Java 8, the method may to return {@code false} for Java 8
+   * distroless unknown to it.
+   *
+   * @param baseImage the configured base image
+   * @return {@code true} if the base image is equal to one of the known Java 8 distroless images,
+   *     else {@code false}
+   */
+  private static boolean isKnownDistrolessJava8Image(String baseImage) {
+    // TODO: drop "latest" and the likes once it no longer points to Java 8.
```

Comment:
Since there's just `:debug` may as well list it explicitly (I think _like_ would be singular: _the like once they no longer_?)

Reply:
There are "jetty:debug" too. Noticed I missed "jetty:latest".